test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -1212,6 +1212,13 @@ def _validate_argument_spec(self, docs, spec, kwargs, context=None):
                     code='parameter-alias-repeated',
                     msg=msg
                 )
+            if not context and arg == 'state' and data.get('choices'):
+                for bad_state in ['list', 'info']:
```

Comment:
How about something like this?
```suggestion
                bad_states = {'list', 'info'} & set(data_choices)
                for state in bad_states:
                    self.reporter.error(
                        path=self.object_path,
                        code='parameter-state-invalid-choice',
                        msg="Argument 'state' includes the value '%s' as a valid choice" % bad_state,
                    )
```